### PR TITLE
Better hyperparameters for dense networks

### PR DESCRIPTION
**Pull request by mrcslws**
I adapted these from:
- projects/backprop_structure/runs/best_configs/ax_ln_mnist.json
- projects/backprop_structure/runs/best_configs/ax_ln_gsc.json

They were generated by an Ax search (ax_ln_mnist.py, ax_ln_gsc.py)

New accuracies, relative to accuracy in paper:
MNIST: 99.38%, previously 99.31%
GSC: 97.37%, previously 96.37%

These don't account for the Dropout results. We probably should have a different set of hyperparameters for the Dropout network.

### CODE REVIEW

**Comment by subutai** [comment]
Can you move these parameters into separate .cfg files, maybe experiments_ax.cfg? Right now the main cfg params and files are tied to the paper. Once I update the paper, I will update the cfg to include the best params. Thanks.

**Comment by mrcslws** [comment]
...@subutai your patience has been much appreciated 🤦
